include/SDL_video.h

```diff
@@ -697,7 +697,10 @@ extern DECLSPEC Uint32 SDLCALL SDL_GetWindowPixelFormat(SDL_Window * window);
  * in pixels may differ from its size in screen coordinates on platforms with
  * high-DPI support (e.g. iOS and macOS). Use SDL_GetWindowSize() to query the
  * client area's size in screen coordinates, and SDL_GL_GetDrawableSize() or
- * SDL_GetRendererOutputSize() to query the drawable size in pixels.
+ * SDL_GetRendererOutputSize() to query the drawable size in pixels. When this
+ * flag is set, the drawable size *must* be queried after every window event as
+ * it can change in the case of the window being moved between displays in a
+ * mixed-DPI environment.
```

Comment:
I don't know about _every_ window event, but it's definitely worth adding something like "Note that the ratio of window size to drawable size may vary after the window is created, you will need to regularly check the drawable size when major window events occur (such as resizing and moving to a new display)."